packages/drizzle-duckdb-wasm/playground/src/App.vue

```diff
@@ -1,89 +1,258 @@
 <script setup lang="ts">
 import type { DuckDBWasmDrizzleDatabase } from '../../src'
 
-import { useDebounceFn } from '@vueuse/core'
+import { DuckDBAccessMode } from '@duckdb/duckdb-wasm'
+import { DBStorageType } from '@proj-airi/duckdb-wasm'
 import { serialize } from 'superjson'
-import { onMounted, onUnmounted, ref, watch } from 'vue'
+import { computed, onMounted, onUnmounted, ref } from 'vue'
 
 import { drizzle } from '../../src'
+import { buildDSN } from '../../src/dsn'
 import * as schema from '../db/schema'
 import { users } from '../db/schema'
 import migration1 from '../drizzle/0000_cute_kulan_gath.sql?raw'
 
 const db = ref<DuckDBWasmDrizzleDatabase<typeof schema>>()
 const results = ref<Record<string, unknown>[]>()
 const schemaResults = ref<Record<string, unknown>[]>()
-const query = ref(`SELECT 1 + 1 AS result`)
+const isMigrated = ref(false)
 
-onMounted(async () => {
-  db.value = drizzle('duckdb-wasm://?bundles=import-url', { schema })
+const storage = ref<DBStorageType>()
+const path = ref('test.db')
+const logger = ref(true)
+const readOnly = ref(false)
+
+const dsn = computed(() => {
+  return buildDSN({
+    protocol: 'duckdb-wasm:',
```

Comment:
```suggestion
    scheme: 'duckdb-wasm:',
```

Perhaps we should call this [`scheme`](https://developer.mozilla.org/en-US/docs/Web/URI/Reference/Schemes)?

Reply:
Sure thing. It seems the URL is using [`protocol`](https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol) but the URI is using `scheme` 🤣 I will update them with `scheme` anyway.